static/js/stream_popover.js

```diff
@@ -469,8 +471,21 @@ function build_move_topic_to_stream_popover(e, current_stream_id, topic_name) {
             },
         );
     }
-
+    function set_stream_topic_typeahead() {
+        const topic_input = $(".inline_topic_edit");
```

Comment:
This selector is too broad -- we could easily have multiple elements with that class.  I suggest using `$("move_topic_form .inline_topic_edit")`.